<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>471</width>
      <height>331</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>471</width>
      <height>331</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>471</width>
      <height>331</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>19</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>22</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Selamat Datang Kembali</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>50</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #a0a0a0;</string>
        </property>
        <property name="text">
         <string>Silahkan masukkan username dan password</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineUsername">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>120</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    background-color: #a0a0a0;
    border: none;
    text-align: center;
    font-size: 10pt;
    border-radius: 6px;  /* Radius sedang, tidak terlalu bulat */
    margin: 2px 8px;
    color: #ffffff;
    padding-left: 5px;
}


</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>100</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>160</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnLogin">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>230</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: rgb(255, 255, 255);
    border: none;
    text-align: center;
    font-size: 10pt;
    border-radius: 6px;  /* Radius sedang, tidak terlalu bulat */
    margin: 2px 8px;
    color: rgb(0,0,0);
}</string>
        </property>
        <property name="text">
         <string>Masuk</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="linePassword">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>180</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    background-color: #a0a0a0;
    border: none;
    text-align: center;
    font-size: 10pt;
    border-radius: 6px;  /* Radius sedang, tidak terlalu bulat */
    margin: 2px 8px;
    color: #ffffff;
    padding-left: 5px;
}


</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>473</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
